### ISSUE TITLE
Instalation not working

### ISSUE THREAD

**Issue by enstain-tm** [benct/lovelace-xiaomi-vacuum-card#80]
I have tired both installation methods, but when I try to add a custom card to lovelace I get "No card type configured"
![Capture](https://user-images.githubusercontent.com/71406634/133293982-b01714ff-a8bf-4aa9-852c-07926a7366c2.PNG)

**Comment by raptor1jec** [comment]
Did you refresh your browser tab?

**Comment by enstain-tm** [comment]
Yes, refreshed, reinstalled, rebooted. Done everything I could think of. I am still fairly new to HA, but I followed the instructions.

**Comment by igorsantos07** [comment]
Same here. I installed it via HACS, which requests to refresh the browser, and so I did. To no avail. Since then I had numerous other Lovelace components installed / updated / refreshed / rebooted, and it still shows the same error - _no cart type found_.

It also doesn't show in the Add Card UI list.

I can see it seems to have been installed since the files are at `www/community/lovelace-xiaomi-vacuum-card`. While I have no clue where the Lovelace config file is, I can see it listed under Config > Lovelace > Resources, as a JS Module (as `/hacsfiles/lovelace-xiaomi-vacuum-card/xiaomi-vacuum-card.js?hacstag=180229356420`, but no clue what that `/hacsfiles` folder is)
